internal/services/applications/application_resource.go

```diff
@@ -304,6 +417,25 @@ func applicationResource() *schema.Resource {
 				}, false),
 			},
 
+			"single_page_application": {
```

Comment:
and here? or will there be other properties there

Reply:
Yeah I'm sort of anticipating additional properties over time in both the `public_client` and `single_page_application` blocks, so was trying to preempt future deprecations here